debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -50,7 +52,7 @@
 
 /**
  * A component that reads the binlog of a MySQL server, and records any schema changes in {@link MySqlSchema}.
- * 
+ *
```

Comment:
will do, i have my editor (intellij) set to remove all trailing whitespace upon saving. will adjust.